### PR TITLE
Add Pohl Boskamp case

### PR DESCRIPTION
**Pull request by michellusthof**
- [x] Add main text and details
- [x] Add quote
- [x] Add company logo
- [x] Add header image
- [x] Add social image

### CODE REVIEW

**Comment by michellusthof** [comment]
[Preview](https://deploy-preview-1868--defacto-de.netlify.com/cases/pohl-boskamp/)

**Comment by michellusthof** [comment]
@isabelamy: Shouldn't 'Moglichkeiten' in the quote on the right be changed to 'Möglichkeiten'?

File: source/cases/de/pohl-boskamp.html.markdown [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+quote:
+  text: "Durch CAPP LMS hat es einen regelrechten Kulturwandel gegeben. Die Möglichkeiten des digitales Lernens wurden internalisiert und sind nun nicht mehr wegzudenken."
+  person: "Torben Sura"
+  position: "Leiter Qualitätssicherung"
+facts:
+  branch: Pharma
+  url: https://www.pohl-boskamp.de/
+  since: 2018
+  employees: Ca. 500
+tags: "pharma"
```
Comment (RonaldDijkstra) [inline source/cases/de/pohl-boskamp.html.markdown]: There is no tag in cases.

Comment (michellusthof) [reply source/cases/de/pohl-boskamp.html.markdown]: I spotted it in another case, but will remove it here.

Comment (RonaldDijkstra) [reply source/cases/de/pohl-boskamp.html.markdown]: You did? :octocat:

Comment (RonaldDijkstra) [reply source/cases/de/pohl-boskamp.html.markdown]: Haha wtf.

Comment (michellusthof) [reply source/cases/de/pohl-boskamp.html.markdown]: > You did? :octocat:

Yep. In both ekv and fek cases (healthcare). But I guess it makes more sense if you have multiple cases covering the same tag/topic.

**Pull request merged**